test/e2e_node/podresources_test.go

```diff
@@ -838,6 +911,20 @@ var _ = SIGDescribe("POD Resources [Serial] [Feature:PodResources][NodeFeature:P
 
 			_, err = cli.GetAllocatableResources(ctx, &kubeletpodresourcesv1.AllocatableResourcesRequest{})
 			framework.ExpectNoError(err, "GetAllocatableResources() failed err %v", err)
+
+			desc := podDesc{
+				podName: "pod-01",
+				cntName: "cnt-01",
+			}
+			tpd := newTestPodData()
+			tpd.createPodsForTest(ctx, f, []podDesc{
+				desc,
+			})
+			expectPodResources(ctx, 1, cli, []podDesc{desc})
+
+			_, err = cli.Get(ctx, &kubeletpodresourcesv1.GetPodResourcesRequest{PodName: "pod-01", PodNamespace: f.Namespace.Name})
```

Comment:
do we want to check the value in the response here?

Reply:
will add